purchases/src/main/kotlin/com/revenuecat/purchases/models/StoreTransaction.kt

```diff
@@ -96,13 +97,51 @@ data class StoreTransaction(
     val subscriptionOptionId: String?,
 
     /**
-     * The prorationMode used to perform the upgrade/downgrade of this purchase.
+     * The replacementMode used to perform the upgrade/downgrade of this purchase.
      * Null if it was not an upgrade/downgrade or if the purchase was restored.
      * This is not available for Amazon purchases.
      */
-    val prorationMode: ProrationMode?,
+    val replacementMode: ReplacementMode?,
```

Comment:
I wonder if we should still have the `prorationMode` as deprecated, but modify the getter so it calculates the `prorationMode` from `replacementMode`

It's not really a hard upgrade so I don't think it's an issue though

Reply:
I think we should deprecate `val replacementMode` after reading that community post about breaking changes. What do you think?

Reply:
Which getter to do you mean? In this case the deprecation should be taken care of with the newly added (and deprecated) constructor that contains the promotationMode param